### PR TITLE
79 6j registration

### PR DESCRIPTION
**Pull request by claraxuxu**
some corrections for the first design version

### CODE REVIEW

File: app/src/androidTest/java/com/bouboulecorp/thurii/DoubleAuth/TelCheckCodeActivityTest.kt [hunk 1]
```diff
@@ -0,0 +1,103 @@
+package com.bouboulecorp.thurii.DoubleAuth
+
+import androidx.test.core.app.ActivityScenario
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/DoubleAuth/TelCheckCodeActivityTest.kt:3]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
 import android.text.TextWatcher
+import android.text.method.LinkMovementMethod
 import android.view.View
-import android.widget.ArrayAdapter
-import android.widget.Spinner
-import android.widget.EditText
-import android.widget.Button
-import android.widget.AdapterView
-import android.widget.Toast
+import android.widget.*
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:8]: 🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/TelCheckCodeActivity.kt [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+        setKeyListener(fromEditText = editTextFive, backToEditText = editTextFour)
+        setKeyListener(fromEditText = editTextSix, backToEditText = editTextFive)
+    }
+
+    private fun initFocus() {
+        editTextOne.isEnabled = true
+        editTextOne.postDelayed({
+            editTextOne.requestFocus()
+            val inputMethodManager = getSystemService(Context.INPUT_METHOD_SERVICE)
+                    as InputMethodManager
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/TelCheckCodeActivity.kt:89]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (20) (should be 16)

File: app/src/main/java/com/bouboulecorp/thurii/Onboarding/OnboardingActivity.kt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.bouboulecorp.thurii.Onboarding
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Onboarding/OnboardingActivity.kt:1]: 🚫 **[ktlint]** <final-newline> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File must end with a newline (\n)

File: app/src/main/java/com/bouboulecorp/thurii/Onboarding/OnboardingActivity.kt [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.bouboulecorp.thurii.R
+
+class OnboardingActivity: AppCompatActivity() {
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Onboarding/OnboardingActivity.kt:7]: 🚫 **[ktlint]** <colon-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing before ":"

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationGender.kt [hunk 6]
```diff
@@ -10,6 +10,7 @@ import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
 import com.bouboulecorp.thurii.R
 
+
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationGender.kt:13]: 🚫 **[ktlint]** <no-consecutive-blank-lines> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Needless blank line(s)

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt [hunk 7]
```diff
@@ -1,11 +1,14 @@
 package com.bouboulecorp.thurii.DoubleAuth
+import android.content.Intent
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:2]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end

**Review by larroyo3 — APPROVED** [review]

**Review by Neyross — CHANGES_REQUESTED** [review]
les test instrumente fails je pense que tu as pas enleve les anciens et que c'est pour ca

**Review by Neyross — APPROVED** [review]
ca va qu'il y a pas de limite sur le coverage pour le moment mais il va falloir avoir plus les porchaines fois

**Pull request merged**